Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivateForTesting.h

```diff
@@ -65,6 +65,8 @@ struct WKAppPrivacyReportTestingData {
 @property (nonatomic, setter=_setMediaCaptureReportingDelayForTesting:) double _mediaCaptureReportingDelayForTesting WK_API_AVAILABLE(macos(12.0), ios(15.0));
 @property (nonatomic, readonly) BOOL _wirelessVideoPlaybackDisabled;
 
+@property (nonatomic, readonly) NSArray<NSNumber *> *_frameProcessIdentifiers;
```

Comment:
_processIdentifiersForFrames? also should this be like... pairs of frameinfos and pids or something?